src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/CustomerThread/Block/list_header.html.twig

```diff
@@ -0,0 +1,68 @@
+{# **
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * #}
+
+<div class="">
+  <div id="customer_thread_contacts" class="d-flex justify-content-around row">
+    {% set nb_categories = servicesSummary|length %}
+    {% for serviceSummary in servicesSummary %}
+
+      <div class="col-lg-3">
+        <div class="card">
+          <div class="card-header">
+            {{ serviceSummary.name }}
+          </div>
+          <div class="card-body">
+            <p>{{ serviceSummary.description }}</p>
+            {% if serviceSummary.getTotalThreads() == 0 %}
```

Comment:
One of the Twig feature is to access the getters easily without relying on method names directly but more like object properties, so you should use this feature which makes the template more readable (to replace everywhere in this file)

```suggestion
            {% if serviceSummary.totalThreads == 0 %}
```

Reply:
Reminder (so you easily find what's left to handle)